src/efjm.tw2

```diff
@@ -35,17 +36,26 @@ Twee2::build_config.story_format = 'Harlowe2'
 ::EFJM - Credits
 Thanks for playing our weird game!
 
-**Created by**
-Nick Cummings
-Nathan Harrison
-Ben Morgan
+A game by Nick Cummings, Ben Morgan, and Nathan Harrison
+
+Illustrations by Sarah Morgan
+
+Map of Juggalo Mountain and surrounding territories by Nick Cummings
+
+Playtesting by Yelena Kolova and Dan Phipps
+
+Special thanks to:
+Sean Dunne
```

Comment:
Could we put a dash before each of these names? It's safer than an HTML list, but it might help them stand out more.

Reply:
Also, I'd vote that we put the names of people we actually know before the names of the people we don't, or make it alphabetical. Either works for me.